tests/child_chain/test_child_chain.py

```diff
@@ -141,3 +144,25 @@ def test_apply_deposit(child_chain):
 
     # Deposit block only contains one transaction
     assert len(child_chain.blocks[deposit_block_number].transaction_set) == 1
+
+
+def test_apply_exit(child_chain):
+    (blknum, txindex, oindex) = (1, 0, 0)
+    sample_event = {
+        'args': {
+            'exitor': '0xfd02EcEE62797e75D86BCff1642EB0844afB28c7',
+            'utxoPos': pack_utxo_pos(blknum, txindex, oindex),
+            'amount': 100,
+        },
+        'event': 'ExitStarted',
+        'logIndex': 0,
+        'transactionIndex': 0,
+        'transactionHash': '0x35e6446818b53b2c4537ebba32b9453b274286ffbb25e5b521a6b0a33e2cb953',
+        'address': '0xA3B2a1804203b75b494028966C0f62e677447A39',
+        'blockHash': '0x2550290dd333ea2876539b7ba474a804a9143b0d4ecb57b9d824f07ffd016747',
+        'blockNumber': 1
+    }
+    child_chain.apply_exit(sample_event)
+
+    # Transaction now marked spent
+    assert child_chain.blocks[blknum].transaction_set[txindex].spent1
```

Comment:
Can you add an assert verifying that the output that's now spent was not before `apply_exit`?